src/cpp/r/CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ target_link_libraries(rstudio-r rstudio-core)
 if(APPLE)
    target_link_libraries(rstudio-r "-undefined dynamic_lookup")
 else()
-   target_link_libraries(rstudio-r "${LIBR_LIBRARIES}")
+   target_link_libraries(rstudio-r ${LIBR_LIBRARIES} ${YAML_CPP_LIBRARIES})
```

Comment:
Why was YAML_CPP_LIBRARIES added here?

Reply:
The Linux build was complaining about a lack of yaml-cpp; my only guess is that this is somehow related to the version of CMake I was using, but it shouldn't hurt at least...